@navikt/core/react/src/form/form-summary/FormSummary.stories.tsx

```diff
@@ -0,0 +1,372 @@
+import { Meta, StoryFn, StoryObj } from "@storybook/react";
+import React from "react";
+import { LinkIcon } from "@navikt/aksel-icons";
+import { VStack } from "../../layout/stack";
+import { Link } from "../../link";
+import FormSummary from "./FormSummary";
+
+const meta: Meta<typeof FormSummary> = {
+  title: "ds-react/FormSummary",
+  component: FormSummary,
+  decorators: [
+    (story) => (
+      <div style={{ width: 600, maxWidth: "100%", margin: "0 auto" }}>
+        {story()}
+      </div>
+    ),
+  ],
+  parameters: { layout: "padded", chromatic: { disable: true } },
+};
+export default meta;
+
+export const Default: StoryFn<typeof FormSummary> = () => (
+  <FormSummary>
+    <FormSummary.Header>
+      <FormSummary.Heading>Personalia</FormSummary.Heading>
+      <FormSummary.Edit />
```

Comment:
Mangler href, bør det være required (kanskje tricky siden den er overridable..)? (gjør det vanskelig å teste med skjermleser nå da den bugger seg uten)

Reply:
Foreslår å endre typen til dette:
```ts
export interface FormSummaryEditProps
  extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
  /**
   * URL to the relevant part of the form, where the answers can be edited.
   */
  href: string;
  /**
   * Link text
   * @default "Endre svar"
   */
  children?: React.ReactNode;
}
```

Reply:
Nå er det bare `Partial<LinkProps>;`
Tenker at man uansett ikke bør kunne endre variant og slike ting.

Reply:
Fungerer dette med f.eks React router `<FormSummary.Edit as={Link} to="#123" />` ?

Reply:
Nei... 🤦‍♂️

Reply:
Hvis vi *virkelig* vil kan vi gjøre dette: `type FormSummaryEditProps = FormSummaryEditBaseProps & ({ as: React.ElementType } | { href: string });` Altså tvinge `href` hvis man ikke bruker `as`.
Men feilmeldingen man får fra TS blir litt kryptisk...
Men kanskje axe og sånn uansett vil fange opp `<a>` uten `href`? Sikkert ikke alle som bruker det da men...

Reply:
Vi kan eventuelt rename Edit til EditLink, slik at det er tydeligere at det er en lenke?

Reply:
> Men kanskje axe og sånn uansett vil fange opp <a> uten href? Sikkert ikke alle som bruker det da men...
Regner med man fanger det opp ganske raskt hvis href ikke er satt riktig, så går sikkert greit at href er optional 🤞